src/App.jsx

```diff
@@ -1,35 +1,35 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import "./App.css";
+import Home from "./components/Home";
+import Navbar from "./components/Navbar";
+import Header from "./components/Header";
+import { Routes, Route } from "react-router-dom";
+import ArticlesList from "./components/ArticlesList";
+import { getArticles } from "./utils/getFunctions";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [allArticles, setAllArticles] = useState([]);
```

Comment:
while you are grabbing the articles-  a visual indication that something is loading would be nice for the user :)